build.sbt

```diff
@@ -220,6 +220,54 @@ lazy val `example-routeguide-client` = project
   .settings(addCommandAlias("runClientTask", "runMain example.routeguide.client.task.ClientAppTask"))
   .disablePlugins(ScriptedPlugin)
 
+////////////////////
+////  TODOLIST  ////
+////////////////////
+
+lazy val `example-todolist-protocol` = project
+  .in(file("modules/examples/todolist/protocol"))
+  .dependsOn(client)
+  .settings(moduleName := "frees-rpc-example-todolist-protocol")
+  .disablePlugins(ScriptedPlugin)
+
+lazy val `example-todolist-runtime` = project
+  .in(file("modules/examples/todolist/runtime"))
+  .settings(moduleName := "frees-rpc-example-todolist-runtime")
+  .settings(exampleTodolistRuntimeSettings)
+  .disablePlugins(ScriptedPlugin)
+
+lazy val `example-todolist-common` = project
```

Comment:
Is really needed?

Reply:
Not needed, I have removed it in next commits